elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/MappingDsl.scala

```diff
@@ -80,8 +80,9 @@ trait MappingDsl extends DslCommons {
 
   case class CompletionMapping(context: Map[String, CompletionContext]) extends FieldMapping {
     val _type = "type" -> "completion"
-    val _analzyer = "analyzer" -> "keyword"
-    val _sanalyzer = "search_analyzer" -> "keyword"
+    // simple analyzer does case insensitive autocomplete
+    val _analzyer = "analyzer" -> "simple"
```

Comment:
this should probably be an option rather than a forced change. Best to me seems to be a case insensitive flag to `CompletionMapping`. It should default to false so that we maintain backwards compatibility.